CHANGELOG/CHANGELOG-1.3.md

```diff
@@ -0,0 +1,95 @@
+- Remove the restriction that a ClusterGroup must exist before it can be used in an Antrea ClusterNetworkPolicy. ([#2478](https://github.com/antrea-io/antrea/pull/2478), [@Dyanngg] [@abhiraut])
+- Remove "controlplane.antrea.tanzu.vmware.com/v1beta1" API as per our API deprecation policy. ([#2528](https://github.com/antrea-io/antrea/pull/2528) [#2631](https://github.com/antrea-io/antrea/pull/2631), [@luolanzone])
+- Controller responses to ClusterGroup membership queries ("/clustergroupmembers" API) now include the list of IPBlocks when appropriate. ([#2577](https://github.com/antrea-io/antrea/pull/2577), [@Dyanngg] [@abhiraut])
+- Install all Endpoint flows belonging to a Service via a single OpenFlow bundle, to reduce flow installation time when the Agent starts. ([#2476](https://github.com/antrea-io/antrea/pull/2476), [@tnqn])
+- Improve the batch installation of NetworkPolicy rules when the Agent starts: only generate flow operations based on final desired state instead of incrementally. ([#2479](https://github.com/antrea-io/antrea/pull/2479), [@tnqn] [@Dyanngg])
+- Use GroupMemberSet.Merge instead of GroupMemberSet.Union to reduce CPU usage and memory footprint in the Agent's policy controller. ([#2467](https://github.com/antrea-io/antrea/pull/2467), [@tnqn])
+- When checking for the existence of an iptables chain, stop listing all the chains and searching through them; this change reduces the Agent's memory footprint. ([#2458](https://github.com/antrea-io/antrea/pull/2458), [@tnqn])
+- Tolerate more failures for the Agent's readiness probe, as the Agent may stay disconnected from the Controller for a long time in some scenarios. ([#2535](https://github.com/antrea-io/antrea/pull/2535), [@tnqn])
+- Remove restriction that only GRE tunnels can be used when enabling IPsec: VXLAN can also be used, and so can Geneve (if the Linux kernel version for the Nodes is recent enough). ([#2489](https://github.com/antrea-io/antrea/pull/2489), [@luolanzone])
+- Automatically perform deduplication on NetworkPolicy audit logs for denied connections: all duplicate connections received within a 1 second buffer window will be merged and the corresponding log entry will include the connection count. ([#2294](https://github.com/antrea-io/antrea/pull/2294) [#2578](https://github.com/antrea-io/antrea/pull/2578), [@qiyueyao])
+- Support returning partial supportbundle results when some Nodes fail to respond. ([#2399](https://github.com/antrea-io/antrea/pull/2399), [@hangyan])
+- When listing NetworkPolicyStats through the Controller API, return an empty list if the `NetworkPolicyStats` Feature Gate is disabled, instead of returning an error. ([#2386](https://github.com/antrea-io/antrea/pull/2386), [@PeterEltgroth])
+- Update OVS version from 2.14.2 to 2.15.1: one interesting fix is to Geneve tunnel support in the userspace datapath (used for Kind clusters). ([#2515](https://github.com/antrea-io/antrea/pull/2515), [@antoninbas])
```

Comment:
is to -> is

Reply:
I don't think that's correct, but I rephrased